src/views/RuleEngine/components/ResourceDialog.vue

```diff
@@ -58,7 +58,10 @@
             :span="item.type === 'object' || item.type === 'array' || item.$attrs.type === 'textarea' ? 24 : 12"
             :key="index"
           >
-            <el-form-item :prop="`config.${item.prop}`">
+            <el-form-item
+              v-if="item.type !== 'file' && !['verify', 'tls_version', 'ciphers'].includes(item.key)"
```

Comment:
diverge from enterprise; we have the value `ciphers` now.
https://github.com/emqx/emqx/blob/3e9abbe95a1b087e7ae0280ad0fc6d4a118d9a16/apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_actions.erl#L227

Reply:
OK